packages/pluggableWidgets/video-player-native/src/ui/Styles.ts

```diff
@@ -1,30 +1,56 @@
 import { Style } from "@mendix/piw-native-utils-internal";
 import { TextStyle, ViewStyle } from "react-native";
 
-interface VideoStyle extends Style {
+export interface VideoStyle extends Style {
     container: ViewStyle;
     indicator: {
         color: string;
     };
     video: ViewStyle;
     errorMessage: TextStyle;
+    fullScreenVideoPlayer: ViewStyle;
+    controlBtnContainerStyle: ViewStyle;
+    fullScreenVideoStyle: ViewStyle;
+    fullScreenActivityIndicatorStyle: ViewStyle;
 }
 
 export const defaultVideoStyle: VideoStyle = {
     container: {
         justifyContent: "center",
         alignItems: "center",
-        backgroundColor: "black",
-        aspectRatio: 16 / 9
+        backgroundColor: "black"
     },
     indicator: {
         color: "white"
     },
     video: {
         width: "100%",
-        height: "100%"
+        height: undefined
     },
     errorMessage: {
         color: "white"
+    },
+    fullScreenVideoPlayer: {
```

Comment:
It's good to add the new styling properties [Atlas 2](https://github.com/mendix/Atlas-UI-Framework/blob/develop/styles/native/ts/core/widgets/videoplayer.ts)

Reply:
done

Reply:
Atlas 2 is version controlled in a separate repository, unfortunately. See the link above.

Reply:
done